src/System.ServiceModel.UnixDomainSocket/src/System/ServiceModel/UnixDomainSocketBinding.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ServiceModel.Channels;
+using System.Xml;
+
+namespace System.ServiceModel
+{
+    public class UnixDomainSocketBinding : Binding
+    {
+        //private OptionalReliableSession _reliableSession;
+        // private BindingElements
```

Comment:
This binding is to provide similar capabilities to NetNamedPipe so won't ever support reliable sessions. Remove the commented out reliable sessions fields.